public/resources/stylesheets/page/_admin.scss

```diff
@@ -81,6 +90,7 @@
   font-weight: 600;
   background-image: linear-gradient(180deg, #af323e 0%, #af323e 100%);
```

Comment:
save the colors in variables, and check if that color are not already in the variables file